packages/select/src/components/select/select2.tsx

```diff
@@ -179,11 +186,15 @@ export class Select2<T> extends AbstractPureComponent2<Select2Props<T>, Select2S
                 onOpened={this.handlePopoverOpened}
                 onOpening={this.handlePopoverOpening}
                 popoverClassName={classNames(Classes.SELECT_POPOVER, popoverProps.popoverClassName)}
+                popupKind="listbox"
```

Comment:
hmm that's unfortunate that you can't use a string literal... I will look into that. in the meantime you can import the `PopupKind` enum.